src/macaron/slsa_analyzer/git_url.py

```diff
@@ -327,25 +327,17 @@ def clone_remote_repo(clone_dir: str, url: str) -> Repo | None:
     # Macaron attempting to clone a repository multiple times.
     # In these cases, we should not error since it may interrupt the analysis.
     if os.path.isdir(clone_dir):
-        logger.info("Checking if the repo %s needs cloning.", url)
         try:
             os.rmdir(clone_dir)
-            logger.info(
-                "The clone dir %s is empty. It has been deleted for cloning the repo %s.",
-                clone_dir,
-                url,
-            )
+            logger.debug("The clone dir %s is empty. It has been deleted for cloning the repo.", clone_dir)
         except OSError:
-            logger.info(
-                "The clone dir %s is not empty. This probably means the repo %s has already been clone. "
+            logger.debug(
+                "The clone dir %s is not empty. This probably means the repo has already been clone. "
                 "No cloning is proceeded.",
```

Comment:
```suggestion
                "The clone dir %s is not empty. Cloning will not be proceeded.",
```

Reply:
Removed in https://github.com/oracle/macaron/pull/352/commits/0f6ee7933caa7609670f73c4cf40d149c35b2f5a